android-project/app/src/main/java/org/libsdl/app/HIDDeviceManager.java

```diff
@@ -108,7 +108,9 @@ private HIDDeviceManager(final Context context) {
         HIDDeviceRegisterCallback();
 
         mSharedPreferences = mContext.getSharedPreferences("hidapi", Context.MODE_PRIVATE);
-        mIsChromebook = mContext.getPackageManager().hasSystemFeature("org.chromium.arc.device_management");
+        mIsChromebook = mContext.getPackageManager().hasSystemFeature("org.chromium.arc.device_management")
```

Comment:
Maybe this should just call the static function in SDLActivity.java?

Reply:
I thought maybe HID could be used without SDLActivity ?

Reply:
It's a static function, so you should be able to call it regardless of whether any activity is instantiated.

Reply:
@slouken updated !